HandsOn/Group11/mappings/RML_operations-with-links.rml

```diff
@@ -0,0 +1,169 @@
+@prefix rr: <http://www.w3.org/ns/r2rml#> .
+@prefix rml: <http://semweb.mmlab.be/ns/rml#> .
+@prefix ql: <http://semweb.mmlab.be/ns/ql#> .
+@prefix dct: <http://purl.org/dc/terms/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix dbo: <http://dbpedia.org/ontology/> .
+@prefix dbr: <http://dbpedia.org/resource/> .
+@prefix art: <http://artwork.org/> .
+
+<#ArtworkTriplesMap>

```

Comment:
I know it's the same as in slides, but rml mapper does not accept this syntax. It expects this to be IRI. So to generate linked data using rml mapper it must be modified. But it's the same in slides so I think we can leave it like that.